database/mssql/scripts/versions/v_28_ddl.sql

```diff
@@ -592,7 +592,7 @@ EXEC sys.sp_addextendedproperty
    @level2type=N'COLUMN',
    @level2name=N'DATE'
 EXEC sys.sp_addextendedproperty 
-   @name=N'MS_Description', 
+   @name=N'MS_Description',
```

Comment:
FYI
v_28 is already in prod. Since the change is limited to removing blank spaces, this should not make any impact and is safe to be in v28